src/components/Delimiter.tsx

```diff
@@ -0,0 +1,25 @@
+import classnames, {
+  display,
+  fontSize,
+  lineHeight,
+  textColor,
+  width,
+} from 'classnames/tailwind'
+
+const bottomSeparator = classnames(
+  width('w-fit'),
+  display('hidden', 'sm:block')
+)
+const statusText = classnames(
+  fontSize('text-xs'),
+  lineHeight('leading-4'),
```

Comment:
`text-xs` should have it by default:
![image](https://user-images.githubusercontent.com/49658988/181080390-c3eec2f3-0c5a-4a38-81c4-8967f63988e4.png)

Reply:
Also, should we extract this into `Text.tsx`?

Reply:
Updated it